### ISSUE TITLE
is this a manual map injector?

### ISSUE THREAD

**Issue by cutypie2351** [leo4048111/Potato-Injector#7]
is this a manual map injector or load lib injector?

**Comment by leo4048111** [comment]
Yes, as I have stated in Readme.md, this injector uses `blackbone::Process::mmap().MapImage`(which is a widely used manual map implementation) to inject dll bytecode.
